gateway/src/apicast/configuration.lua

```diff
@@ -140,20 +142,48 @@ function _M.services_limit()
 end
 
 function _M.filter_services(services, subset)
-  subset = subset and util.to_hash(subset) or _M.services_limit()
-  if not subset or not next(subset) then return services end
-
   local s = {}
+  local serviceRegexpFilter  = env.value("APICAST_SERVICES_FILTER_BY_URL")
```

Comment:
The project uses snake_case. Can you change this to be consistent with the rest of the project?